src/Accelerators/NNPA/Runtime/zDNNExtension/zDNNExtension.c

```diff
@@ -147,10 +250,13 @@ static void freeZTensorChunk(zdnn_ztensor *t) {
 
 static void copyZTensorChunk(
     const SplitInfo *splitInfo, uint32_t chunkID, bool fromChunk) {
-  // Only support the second innermost axis in the CPU tensor at this moment.
-  // axis = 2 in the CPU tensor corresponds to dim3 in zTensor.
-  if (splitInfo->axis != 2) {
-    printf("Only support the second innermost dimension at this moment.");
+  // Only support the first and second innermost axis in the CPU tensor at this
```

Comment:
Again, for OpenMP, would be convenient if you could define what is modified here... I think its only the private data in a single chunkID when copying to chunk, and when copying from chunk it copes into the shared "origTensor" but each buffer is guaranteed (if that is true) to write only into a distinct tile than any other chunks.